### ISSUE TITLE
Done category

### ISSUE THREAD

**Issue by VisnuRavi** [VisnuRavi/pe#4]
After a entry is completed, we enter done identifier, which marks the entry as done and puts it in the done category. However,  once an entry is done, it can be deleted away. Keeping it to keep records is not very useful. 

Even if this done feature is deemed to be useful to keep done entries, then there should also be an option to add in done items directly through the command box, instead of needing to first put it into a todo/backlog/progress before marking it as done, especially for first time users who may have already done certain tasks before they started using this application. This way they would be able to keep track of what they have done.
